src/AppInstallerRepositoryCore/Rest/Schema/Json/SearchRequestSerializer.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "pch.h"
+#include "cpprest/json.h"
+#include "Rest/Schema/IRestClient.h"
+
+namespace AppInstaller::Repository::Rest::Schema::Json

```

Comment:
Shouldn't these serialize/deserialize functions be in the 1_0 directory?  They seem like the primary thing that would change between versions.

Reply:
I agree, I will move them to 1_0 dir